package.json

```diff
@@ -43,6 +43,7 @@
     "extends": [
       "plugin:shopify/node",
       "plugin:shopify/esnext",
```

Comment:
esnext should be above node as esnext is a "core" config while "node" is an extension, per https://github.com/shopify/eslint-plugin-shopify#provided-configurations

Reply:
Thanks! I had no idea plugins were order-sensitive.